docs/basic/usage.rst

```diff
@@ -236,3 +236,55 @@ integration between your Python program and your PostgreSQL database:
 - If you want to customise how Python values and PostgreSQL types are mapped
   into each other, beside the :ref:`basic type mapping <types-adaptation>`,
   you can :ref:`configure your types <adaptation>`.
+
+
+.. _logging:
+
+Connection logging
+------------------
+
+Psycopg uses the `logging` module to report the operations happening at
```

Comment:
Is it worth clarifying that this is the _standard library_ logging module (as opposed to a psycopg internal logging module)?

Is it easy to link to https://docs.python.org/3/library/logging.html

Reply:
The above links to the stdlib docs indeed (see [rendered version](https://www.psycopg.org/psycopg3/docs/basic/usage.html#connection-logging)). No hurt in making it
```
uses the stdlib `logging` module"
```
though. Thank you!

Reply:
Ah nice, is that Sphinx magic? (I'm big on documentation, in general, but am a complete Sphinx newb!)

Reply:
Yes, "[intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html)"

https://github.com/psycopg/psycopg/blob/48a151b805224f090577341e7809cd7fc9fcc3a3/docs/conf.py#L95-L99